doc/scenario.md

```diff
@@ -852,3 +853,17 @@ Input facts (set by config):
 
 - `cartridge_runtime_params` - new instance parameters ([more details here](/doc/instances.md));
 - `expelled` - indicates if instance must be expelled from topology.
+
+### cleanup_instance
```

Comment:
`clanup_expelled = stop_service + cleanup_instance_files`
`cleanup_instance =  cleanup_instance_files`


Maybe `cleanup_instance_files` is a better name?